src/signals/incident/components/form/ContainerSelect/Selector/Selector.test.tsx

```diff
@@ -15,47 +15,74 @@ describe('signals/incident/components/form/ContainerSelect/Selector', () => {
   beforeEach(() => {
     fetchMock.resetMocks();
     fetchMock.mockResponseOnce(JSON.stringify(containersJson), { status: 200 });
+    showDesktopVariant = false;
   });
 
   afterEach(() => {
     jest.resetAllMocks();
   });
 
-  it('should render the component', async () => {
+  it('should render the component', () => {
     render(withContainerSelectContext(<Selector />));
 
-    expect(await screen.findByText(/container toevoegen/i)).toBeInTheDocument();
-    expect(screen.getByText(/container verwijderen/i)).toBeInTheDocument();
-    expect(screen.getByText(/meld deze container\/sluiten/i)).toBeInTheDocument();
+    expect(screen.getByTestId('containerSelectSelector')).toBeInTheDocument();
   });
 
-  it('should call update when adding container', async () => {
+  it('should call update when adding container', () => {
     render(withContainerSelectContext(<Selector />));
     expect(contextValue.update).not.toHaveBeenCalled();
 
-    const element = await screen.findByText(/container toevoegen/i);
-    if (element) fireEvent.click(element);
+    const addContainerButton = screen.getByText(/containers toevoegen/i);
```

Comment:
The async construction was added to avoid some nasty error in the console. Why is it removed? 
![image](https://user-images.githubusercontent.com/24289432/105482209-5b3e4200-5ca8-11eb-93c0-2b7fd1bcf1a6.png)

Reply:
Fixed